publiccode.yml

```diff
@@ -16,10 +16,21 @@ categories:
   - data-analytics
   - data-collection
   - data-visualization
+  - open-data
```

Comment:
Purtroppo la chiave `categories` è chiusa e la lista delle possibili voci è questa: https://docs.italia.it/italia/developers-italia/publiccodeyml/it/master/categories-list.html#tag-validi
Per `data` ho trovato solo quelle 3 e non `open-data` e dunque dobbiamo rimuoverlo